tests/integration/ReactionTest.php

```diff
@@ -0,0 +1,156 @@
+use GetStream\Stream\Feed;
+use PHPUnit\Framework\TestCase;
+use Ramsey\Uuid\Uuid;
+
+class ReactionTest extends TestCase
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * @var Feed
+     */
+    protected $user1;
+
+    /**
+     * @var Feed
+     */
+    protected $user2;
+
+    /**
+     * @var Feed
+     */
+    protected $aggregated2;
+
+    /**
+     * @var Feed
+     */
+    protected $aggregated3;
+
+    /**
+     * @var Feed
+     */
+    protected $flat3;
+
+    /**
+     * @var string
+     */
+    protected $activity_id;
+
+    /**
+     * @var Reactions
+     */
+    protected $reactions;
+
+    protected function setUp()
+    {
+        $this->client = new Client(
+            getenv('STREAM_API_KEY'),
+            getenv('STREAM_API_SECRET'),
+            'v1.0',
+            getenv('STREAM_REGION')
+        );
+        $this->client->setLocation('qa');
+        $this->client->timeout = 10000;
+        $this->user1 = $this->client->feed('user', Uuid::uuid4());
+        $this->user2 = $this->client->feed('user', Uuid::uuid4());
+        $this->aggregated2 = $this->client->feed('aggregated', Uuid::uuid4());
+        $this->aggregated3 = $this->client->feed('aggregated', Uuid::uuid4());
+        $this->flat3 = $this->client->feed('flat', Uuid::uuid4());
+        $activity_data = ['actor' => 1, 'verb' => 'tweet', 'object' => 1];
+        $response = $this->user1->addActivity($activity_data);
+        $this->activity_id = $response['id'];
+        $this->reactions = $this->client->reactions();
+    }
+
+    public function testSimpleAddReaction()
+    {
+        $reaction = $this->reactions->add('like', $this->activity_id, 'bob');
+        $this->assertSame($reaction['user_id'], 'bob');
+        $this->assertSame($reaction['kind'], 'like');
+        $this->assertSame($reaction['activity_id'], $this->activity_id);
+    }
+
+    public function testAddDataReaction()
+    {
+        $data = array('client' => 'php');
+        $reaction = $this->reactions->add('like', $this->activity_id, 'bob', $data);
+        $this->assertSame($reaction['user_id'], 'bob');
+        $this->assertSame($reaction['kind'], 'like');
+        $this->assertSame($reaction['activity_id'], $this->activity_id);
+        $this->assertSame($reaction['data'], $data);
+    }
+
+    public function testAddChildReaction()
+    {
+        $initial_reaction = $this->reactions->add('like', $this->activity_id, 'bob');
+        $child_reaction = $this->reactions->addChild('like', $initial_reaction['id'], 'alice');
+        $this->assertSame($child_reaction['user_id'], 'alice');
+        $this->assertSame($initial_reaction['user_id'], 'bob');
+        $this->assertSame($child_reaction['kind'], 'like');
+        $this->assertSame($child_reaction['activity_id'], $this->activity_id);
+        $this->assertSame($child_reaction['parent'], $initial_reaction['id']);
+    }
+
+    public function testAddTargetFeedsReaction()
```

Comment:
Maybe this test or a separate one could check for the format of the reaction posted to the target feed